cirq/aqt/aqt_simulator_test.py

```diff
@@ -0,0 +1,9 @@
+from nose.tools import raises
```

Comment:
Instead of nose, use pytest.

Example: https://github.com/quantumlib/Cirq/blob/bda19f7bb9e59ad7079c613aa16607cea2ae2b39/cirq/google/engine/engine_test.py#L143

Reply:
Thank you for the help. The local pylint always crashes and mypy shows not all issues.

Reply:
Are these the versions you are using? https://github.com/quantumlib/Cirq/blob/master/dev_tools/conf/pip-list-dev-tools.txt